topics/go/algorithms/strings/permuation/permutation.go

```diff
@@ -0,0 +1,25 @@
+package strings
+
+import (
+	"sort"
+	"strings"
+)
+
+// IsPermutation check is two strings are permutations.
+func IsPermutation(str1, str2 string) bool {
+
+	// If the length are not equal they cannot be permutation.
+	if len(str1) != len(str2) {
+		return false
+	}
+
+	// Create slices for each input string.
+	s1 := strings.Split(str1, "")
```

Comment:
Describe what this does better.

Are you breaking each character out? Describe what this is doing?

Reply:
Fixed